source/refunding_payments/index.html.md.erb

```diff
@@ -13,6 +13,8 @@ You can also refund a payment by signing into the [GOV.UK Pay admin tool](https:
 
 Your user will receive the refund in the same account they originally paid from. You cannot refund a payment to another card or bank account.
 
+You can refund payments regardless of whether you [integrate with the GOV.UK Pay API](/integrate_with_govuk_pay/#integrate-with-the-gov-uk-pay-api) or use [payment links](https://www.payments.service.gov.uk/govuk-payment-pages/).
```

Comment:
I don't know that we need to say this. Maybe just calling out where it is different for payment links is sufficient?

Reply:
ok will remove